addons/skin.estuary/xml/Includes_PVR.xml

```diff
@@ -213,7 +213,7 @@
 					<top>60</top>
 					<height>200</height>
 					<wrapmultiline>true</wrapmultiline>
-					<label>$INFO[ListItem.TimerType,[COLOR grey]$LOCALIZE[803]:[/COLOR] ,[CR]]$INFO[ListItem.Duration,[COLOR grey]$LOCALIZE[180]:[/COLOR] ,[CR]]$VAR[ExpirationDateTimeLabel,[CR]]$VAR[FlagLabel]</label>
+					<label>$INFO[ListItem.TimerType,[COLOR grey]$LOCALIZE[803]:[/COLOR] ,[CR]]$VAR[DateDurationLabel,[CR]]$VAR[ExpirationDateTimeLabel,[CR]]$VAR[FlagLabel]</label>x
```

Comment:
stray `x` at the end of the line

don't think there's a need to include a `[CR]` in the $VAR?
it's already included in the individual infolabels.

likely `$VAR[ExpirationDateTimeLabel,[CR]]` doesn't need it either, could you please double-check?

Reply:
Not sure where the `x` came from!

If I remove the CRs nothing displays. No date/duration/progress bar. It's all blank.

Reply:
So I just removed the X

Reply:
> If I remove the CRs nothing displays.

that shouldn't happen... please give this a shot:

`<label>$INFO[ListItem.TimerType,[COLOR grey]$LOCALIZE[803]:[/COLOR] ,[CR]]$VAR[DateDurationLabel]$VAR[ExpirationDateTimeLabel]$VAR[FlagLabel]</label>`

Reply:
Sorry, you are right. I got mixed up in square brackets!

PR updated ;)

Reply:
Is this ok now @ronie ?